docs/version-1.9/modules/en/pages/upgrades/longhorn-components/upgrade-longhorn-manager.adoc

```diff
@@ -1,29 +1,35 @@
 = Upgrade Longhorn Manager
 :current-version: {page-component-version}
 
+[WARNING]
+====
+It is recommended to create a xref:snapshots-backups/system-backups/create-system-backup.adoc[Longhorn system backup] before performing the upgrade. This ensures that all critical resources, such as volumes and backing images, are backed up and can be restored in case any issues arise.
+====
+
 == Upgrading from v1.8.x
 
 {longhorn-product-name} only supports upgrading to v{current-version} from v1.8.x. For other versions, please upgrade to v1.8.x first.
 
 {longhorn-product-name} v1.8.x to v{current-version} support live engine upgrades.
```

Comment:
Remove this because this doc is talking about longhorn-manager rather than longhorn engine